### PR TITLE
[ADD] Added the OCA Contribution Repository Policy

### PR DESCRIPTION
**Pull request by elicoidal**
@max3903 
cc @OCA/board

### CODE REVIEW

**Comment by pedrobaeza** [comment]
As said, :-1:

File: website/Contribution/oca_repository_policy.rst [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+.. |CATEGORIES| replace:: Contribution, Repository Management, Policy
+
+.. fill in the Document information in the above lines. It will be automatically
+   replaced in the document header. Don't forget to remove the []
+
+.. contents:: Table of contents
+    :depth: 4
+
+.. sectnum::
+
```
Comment (sbidoul) [inline website/Contribution/oca_repository_policy.rst]: @elicoidal Can you insert a word saying the canonical URL to this document is https://odoo-community.org/page/development-status

I'll adapt the redirection when it's merged.

**Comment by elicoidal** [comment]
@OCA/board

File: website/Contribution/oca_repository_policy.rst (hunk 1 above)
Comment (sbidoul) [reply website/Contribution/oca_repository_policy.rst]: @elicoidal sorry, my above comment does not make sense, I though it was the development status document (maturity level), but now I see there is no PR for that one. https://odoo-community.org/page/development-status

Comment (elicoidal) [reply website/Contribution/oca_repository_policy.rst]: I will remove the footer and reference to this link

File: website/Contribution/oca_repository_policy.rst [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+Review Guidelines [TODO]
+------------------------
+
+At the feature curation level, review guidelines are needed to help
+reviewers decide on the adequate maturity levels, and steps needed to
+bring the module to the next maturity level.
+
+.. note::
+   The canonical URL to this document is https://odoo-community.org/page/repository-policy.
```
Comment (elicoidal) [inline website/Contribution/oca_repository_policy.rst]: @sbidoul please review again. I have added this URL

**Comment by elicoidal** [comment]
can we merge this? @OCA/board

**Pull request merged**